FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/startup.c

```diff
@@ -96,9 +96,19 @@ void prvGetRegistersFromStack( uint32_t * pulFaultStackAddress )
     for( ; ; )
     {
     }
+
+    /* remove the warning: variable <x> is set but not used */
+    ( void ) r0;
+    ( void ) r1;
+    ( void ) r2;
+    ( void ) r3;
+    ( void ) r12;
+    ( void ) lr;
+    ( void ) pc;
+    ( void ) psr;
 }
 
-static void Default_Handler( void ) __attribute__( ( naked ) );
+static __attribute__( ( naked ) )
```

Comment:
Why this change? The attributes are supposed to be added to the declaration - https://gcc.gnu.org/onlinedocs/gcc-4.1.2/gcc/Function-Attributes.html.

Reply:
Ok, kept this code as is.

While this docu suggests that it only works in separate declarations, adding it to function definitions also works.
Quick review shows that current code sometimes also just has function definitions with this, no separate
function declarations.

Starting point for me was that one such function declatation had the wrong name, so an updated patch just fixes this one typo now.